tests/script_jackrabbit.sh

```diff
@@ -12,7 +12,7 @@ elif test "${SCRIPT_DIR:0:1}" != "/" ; then
   SCRIPT_DIR="$PWD/$SCRIPT_DIR"
 fi
 
-./tests/jackrabbit.sh
+./tests/jackrabbit.sh || exit
```

Comment:
why this change? should we do `set -e` at the top of the script to abort on any error?